baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ImmutablesStyleCollision.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.baseline.errorprone;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.BugPattern.LinkType;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.matchers.ChildMultiMatcher;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.matchers.Matchers;
+import com.sun.source.tree.ClassTree;
+import org.immutables.value.Value;
+
+@AutoService(BugChecker.class)
+@BugPattern(
+        name = "ImmutablesStyleCollision",
```

Comment:
Please document this alongside the other baseline error-prone checks in the readme!

Reply:
You'll also need to add it to the list of errorprone rules we always apply

Reply:
I added it to the readme but where is the list @ferozco ?

Reply:
I think that's just for automatic refactors, but this can't be auto-fixed. `severity = BugPattern.SeverityLevel.ERROR,` should be sufficient.